src/App.js

```diff
@@ -16,7 +32,7 @@ const DATA = [
   {category: 'Electronics', price: 199.99, stocked: true, name: 'Nexus 7', id: 6}
 ]
 
-function roundUp (num) {
+export function roundUp (num) {
```

Comment:
Nice function. For something like this, I would encourage you to use jsdoc to describe what the function does. To me, it wasn't clear that it rounded up to a penny. A docstring like this would make it more clear:
```javascript
/**
 * Rounds a floating-point number up to the nearest penny. For example: 1.115 -> 1.12.
 * and 1.111 -> 1.11
 * @param num {Number} - any input number
 * @returns {Number} - the rounded value
**/
```